Sources/JXKit/JXConvertible.swift

```diff
@@ -148,17 +148,10 @@ extension CGFloat: JXConvertible {
 }
 #endif
 
-#if canImport(Foundation)
```

Comment:
I found some places where we were already unconditionally importing all Foundation, so I removed all attempts to conditionalize or limit it.